e3db-crypto-android/src/main/java/com/tozny/e3db/AndroidCrypto.java

```diff
@@ -51,7 +51,7 @@ public AndroidCrypto() {
   private static class Init {
     // Static inner class as a singleton to make sure
     public static final SodiumAndroid sodium;
-    // Sodium library is initalized once and only once.
```

Comment:
🤦‍♂️ me, every time. Constructor, now there's a word we can all spell correctly right?